vuestorefront/schemas/product.py

```diff
@@ -108,22 +108,16 @@ def get_product_list(env, current_page, page_size, search, sort, **kwargs):
     else:
         offset = 0
     order = get_search_order(sort)
-    products = Product.search(domain, order=order)
-
-    # If attribute values are selected, we need to get the full list of attribute values and prices
-    if domain == partial_domain:
-        attribute_values = products.mapped('variant_attribute_value_ids')
-        prices = products.mapped('list_price')
-    else:
-        without_attributes_products = Product.search(partial_domain)
-        attribute_values = without_attributes_products.mapped('variant_attribute_value_ids')
-        prices = without_attributes_products.mapped('list_price')
-
+    products = Product.search(domain, order=order, offset=offset, limit=page_size)
+    prices = products.mapped('list_price')
```

Comment:
This will return `max` and `min` prices only for the current batch.  However, IMO finding max and min price as it is implemented now can hit on performance for larger amount of products. Let's add a TODO to reimplement this part with a better solution and don't return any prices for now, we can return 0, 0 for them.

Reply:
What would be normal limit? I hope it would not search for thousands of products at a time, so should not be a problem.

Reply:
At one point they need all products at once so VSF can create routes for it, this is what I was informed from the frontend devs. I guess that would happen only on server start/restart.
However, my intention to remove this was more because IMO is better to not send any data instead of sending misliding data. And current implementation is giving as a result the range for the products on the page not for all products.

Reply:
But usually thats how it should be, you are not supposed to query all possible records. This would return insane amount of data. So thats why I added offset/limit on search (they also returned limited results originally, but they would first search all and then filter...).
So you want for now just return prices as 0 and not look for prices at all?